### ISSUE TITLE
make command showing error in config file

### ISSUE THREAD

**Issue by Swagat47** [FSillT/QCADesigner-E#3]
./configure: 6534: ./configure.lineno: Syntax error: word unexpected (expecting ")")
make: *** [Makefile:309: config.status] Error 2

**Comment by FSillT** [comment]
Hi,
what is your environment? What are your configuration options?

**Comment by Swagat47** [comment]
I'm using windows 11. But I'm using windows subsystem for Linux and using Ubuntu 22.04. I dont understand what do you mean by configuration options but as for the configure.in file is concerned I have not made any changes to that.

Apart from that to run `$ ./autogen.sh` I had to use `chmod +x ./autogen.sh`
After that I tried running `$ ./configure --prefix=/home/your_account/your_install_location` but it gave some syntax error

![Screenshot 2023-09-12 173646](https://github.com/FSillT/QCADesigner-E/assets/55080432/e835124d-4624-4937-88c1-47ab3cfa0152)

I ignored this error and ran the `make` command but it gave this error

![Screenshot 2023-09-12 173657](https://github.com/FSillT/QCADesigner-E/assets/55080432/5107f6da-5ff6-4357-8714-926505b2ed3d)

**Comment by FSillT** [comment]
did you check, whether the indicated folder (/home/your_account/your_install_location) exists? It might be that it has problems to install GTK in your environment. You might install it by hand.

**Comment by Swagat47** [comment]
Yes the indicated folder exists. I'll try to install GTK manually

**Comment by Swagat47** [comment]
config.status: error: cannot find input file: `QCADesigner.desktop.in'

while running the `$ ./configure --prefix=/home/your_account/your_install_location` I have entered the correct location as per my machine. While executing this command it throws this error


![Screenshot from 2023-11-01 23-17-38](https://github.com/FSillT/QCADesigner-E/assets/55080432/7196f148-5fa3-4a9d-9d38-06befb558d6d)

**Comment by FSillT** [comment]
I'd need the complete output, as that file is actually generated during configuration. Though, I'm still somewhat doubtful that your environment is supported. Unfortunately, i can't test it.

**Comment by Swagat47** [comment]
Is the tool not supported on Ubuntu 20.04.6 LTS. Earlier I was trying it on wsl, but now I'm just uisng Linux OS(Ubuntu 20.04.6 LTS).

**Comment by FSillT** [comment]
Ubuntu 20.04.6 LTS is supported. As I said, I'd need the log-files, as the file that can't be found is generated during the make process (see src\Makefile and src\Makefile.in)

**Comment by Swagat47** [comment]
![Screenshot from 2023-11-06 08-58-23](https://github.com/FSillT/QCADesigner-E/assets/55080432/002c0e2e-ee00-4087-ad6c-88767a8094f5)

these error message start at the very first step. I ran `chmod -x ./autogen.sh` because without it, it outputs permission denied. After that I ran the ./autogen.sh command and you can see the above ss for its logs.
Where can I find the log-file you mentioned in the above cooment.

![Screenshot from 2023-11-06 09-34-48](https://github.com/FSillT/QCADesigner-E/assets/55080432/0d19a981-9206-4c03-8c35-b5ec977e4e0b)
which one of these specifically do you require? Is it config.log file
[config.log](https://github.com/FSillT/QCADesigner-E/files/13262353/config.log)
